package.json

```diff
@@ -222,5 +155,159 @@
       "post-commit": "git update-index --again",
       "pre-commit": "lint-staged"
     }
+  },
+  "dependencies": {
+    "bolt-check": "^0.3.0",
+    "@babel/helper-module-imports": "^7.0.0",
```

Comment:
shouldnt those be kept as devDeps? it doesnt matter much for the root package.json but seems like devDep has a better semantic for those

Reply:
Because it doesn't matter and dependencies is the default.